test/safe/of_test.rb

```diff
@@ -27,7 +27,7 @@ def test_neato
 
   class Welp
     def to_s
-      "¯\_(ツ)_/¯"
+      "¯\\_(ツ)_/¯"
```

Comment:
y u no 🤷

Reply:
Lol this was actually just a new Standard fix